components/org.wso2.carbon.identity.conditional.auth.functions.http/src/test/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImplTest.java

```diff
@@ -475,4 +506,32 @@ public Map<String, String> dummyTokenEndpoint(@HeaderParam("Authorization") Stri
         }
         return response;
     }
+
+    @FunctionalInterface
+    public interface ResponseValidator {
+
+        @HostAccess.Export
+        boolean validateResponse(JsParameters response);
+    }
+
+    public class ResponseValidatorImpl implements HTTPPostFunctionImplTest.ResponseValidator {
+
+        @Override
+        @HostAccess.Export
+        public boolean validateResponse(JsParameters response) {
+
+            try {
+                if (response != null) {
+                    JsonObject expectedResponse = sequenceHandlerRunner.loadJson(HTTP_GET_RESPONSE, this);
+                    Gson gson = new Gson();
+                    String dataStr = gson.toJson(response.getWrapped());
+                    JsonObject actualResponse = gson.fromJson(dataStr, JsonObject.class);
+                    return actualResponse.equals(expectedResponse);
+                }
+                return false;
+            } catch (JsTestException e) {
+                throw new RuntimeException(e);
```

Comment:
Check on throwing runtime exception

Reply:
We don't need to throw RuntimeException. Hence addressed with 46c8e19185f03a9c7a8e20a87f85cae4e5e0ea96